x/streamswap/README.md

```diff
@@ -0,0 +1,121 @@
+# Streamswap
+
+## Abstract
+
+Streamswap is a new way and innovative way of selling token sale.
+The mechanism allows anyone to create a new Sale event and sell any
+amount of tokens in a more democratic way than the traditional solutions.
+
+## Context
+
+Since the first ICO boom, token sale mechanism was one of the driving
+force for web3 onboarding.
+Promise of a cheap tokens which can quickly accrue value is very attractive
+for casual any sort of investors. Easy way of fundrising (the funding team)
+opened doors for cohorts of new teams to focus on building on web3.
+
+Traditional mechanisms of token sale included:
+
+- Automated ICO, where team decides about the issuance price and the sale
+  happens through a swap controlled by a smart contract.
+- Regulated, centralized ICO - token sale controlled by a dedicated company,
+  which will preform all operations using centralized services meeting
+  regulatory requirements (KYC...). Example: Coinlist sales.
+- Balancer style ICO: a novel solution to utilize Dutch Auction mechanism to
+  find a fair strike price.
+
+The first two mechanisms are not well suited for early stage startups, where
+the token sale price is usually defined by a founding team and can't be
+impacted by the ecosystem wisdom. False marketing actions are usually setup
+to support their initial price.
+
+The latter mechanism is not democratic - big entities can control the
+price movements or place big orders leaving smaller investors with nothing.
+
```

Comment:
I think it'll be really helpful to add what makes streamswap more "democratic" than the rest

Reply:
It's explained below: anyone can join and has guarantee to received tokens.